CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Change interceptors in `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` to disable `SENT`/`RECEIVED` events.
   Use `WithMessageEvents()` to turn back on. (#3964)
 
+### Changed
+
+- `go.opentelemetry.io/contrib/detectors/gcp`: Detect faas.instance instead of faas.id, since faas.id is being removed. (#4198)
```

Comment:
```suggestion
- `go.opentelemetry.io/contrib/detectors/gcp`: Detect `faas.instance` instead of `faas.id`, since `faas.id` is being removed. (#4198)
```